### DISCUSSION TITLE
naming each alarm

### DISCUSSION THREAD

**Discussion by vishnup3211** [petrleocompel/hikaxpro_hacs#89] [Q&A] [answered]
is it possible to rename the entry since we have multiple alarms and unable to idenify using the  hikvision_axpro_ip adrees format

**Comment by petrleocompel** [comment, marked as answer]
alarms are distinguished by "IP" but then by "name" of the system. you can test and report.. But I was using 2 alarms in one HA. But it kinda does not make sense to mix 2 homes in one HA.
